test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -4584,6 +4584,84 @@ TEST_P(SslSocketTest, RevokedIntermediateCertificateCRLInTrustedCA) {
   testUtil(complete_unrevoked_test_options.setExpectedSerialNumber(TEST_SAN_DNS4_CERT_SERIAL));
 }
 
+TEST_P(SslSocketTest, NotRevokedLeafCertificateOnlyLeafCRLValidation) {
+  // The test checks that revoked certificate will makes the validation fails even if we set
```

Comment:
this test's validation did not fail right? it succeeds. see your line 4611 comment.